libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRouteCallback.java

```diff
@@ -29,8 +29,10 @@
   public void onResponse(@NonNull Call<DirectionsResponse> call, @NonNull Response<DirectionsResponse> response) {
     callback.onResponse(call, response);
     if (isValid(response)) {
-      String uuid = response.body().uuid();
-      sendEventWith(listener.getTime(), uuid);
+      DirectionsResponse directionsResponse = response.body();
+      if (directionsResponse != null) {
+        sendEventWith(listener.getTime(), directionsResponse.uuid());
```

Comment:
wouldn't `directionsResponse.uuid()` returning `null` be problematic here?

Reply:
No, I've checked all calls sequence and in a result this uuid converts to String and sending as some metric event. So I think it is ok here.